.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: groovy
 sudo: false
 jdk:
-- openjdk7
+- oraclejdk8 
```

Comment:
Why did you change this? The plugin needs to work with Grails which is built with Java 7